### ISSUE TITLE
Allow multiplayer games with Custom Ruleset

### ISSUE THREAD

**Issue by poly000** [ppy/osu#11736]
**Describe the new feature:** 

Allow multiplayer games with Custom Ruleset.

**Proposal designs of the feature:** 

Use some sort of signature to identify Custom Rulesets and allow trusted Custom Rulesets to be used in multiplayer games

**Comment by peppy** [comment]
Hmm, I thought we already had an issue open tracking this, but I guess not. Let's keep this one open to track the larger task (it doesn't just apply to multiplayer).

This is a task we definitely want to get to, but it will happen after everything else is in place. Don't expect it too soon.

**Comment by Axilotl17** [comment]
Now that Lazer is close to being ranked, is there any better idea on when this may come to fruition?

**Comment by bdach** [comment]
No. We're not even done with the bundled rulesets (see: editor, among others).

I'd say years away at the minimum.
